cli/lib/kontena/cli/stacks/common.rb

```diff
@@ -110,6 +110,7 @@ def self.included(where)
         where.option '-v', "VARIABLE=VALUE", "Set stack variable values, example: -v domain=example.com. Can be used multiple times.", multivalued: true, attribute_name: :var_option do |var_pair|
           var_name, var_value = var_pair.split('=', 2)
           values_from_value_options.merge!(::YAML.safe_load(::YAML.dump(var_name => var_value)))
+          var_pair
```

Comment:
just to return the correct thing so that `var_option_list` is an array of `-v` values.